### PR TITLE
*P1 - FE - Transaction loaders [5]

### PR DESCRIPTION
**Pull request by Space-zee**
https://app.clickup.com/t/33z3g56

### CODE REVIEW

File: contracts_arbitrum/hardhat.config.ts [hunk 1]
```diff
@@ -21,9 +21,9 @@ const hardhatNetwork = process.env.NO_FORK ?
     deploy: ['deploy/dev']
   }:
   {
-    chainId: 421613,
+    chainId: 42161,
     forking: {
-      url: 'https://arb-goerli.g.alchemy.com/v2/i7XMrv80YbBZuJBt6u0I7zaAmY9gzERx',
+      url: 'https://arb1.arbitrum.io/rpc',
```
Comment (ruslanpalagin) [inline contracts_arbitrum/hardhat.config.ts:26]: please stop changing it, you can use env vars to specify fork. For example:
FORK_MAINNET=1 npx hardhat node
FORK_TESTNET=1 npx hardhat node

File: frontend/app/features/assets/controllers/NewAssetController.tsx [hunk 2]
```diff
@@ -149,18 +149,23 @@ export const NewAssetController = () => {
...
-                      rentalManagementForm={ rentalManagementForm }
-                      onReturnHome={ returnHome } currentForm={ currentForm }
-                      onCurrentFormChange={ setCurrentForm } onClickLocal={ onClickLocal }
-                      onFileUploadLocal={ onFileUploadLocal } onSendAllData={ sendAllData }
-                      sendDataStatus={ sendDataStatus }/>
+      <button onClick={deployNewAsset}>Deploy new asset</button>
+      {address ?
+        <div style={{position:'relative' ,height: '100%'}}>
+          <div style={{position:'absolute', right:10, height: '100%'}}>
+            <TxPopUp txStatus={txStatus} setTxStatus={setTxStatus}/>
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/controllers/NewAssetController.tsx:156]: put it on _app.tsx please. Globally

File: frontend/app/features/assets/models/newAssetModel.ts [hunk 3]
```diff
@@ -137,24 +138,34 @@ export const $listingAssetInBlockchain = atom(null, async (get, set, args: {
...
-  let newAssetABI = ["event NewAsset(address indexed assetAddress)"];
-  let iface = new ethers.utils.Interface(newAssetABI);
-  const event = iface.parseLog(result.logs[0]);
-  return event.args.assetAddress
+  try {
+    set($txStatus, TxStatus.Pending);
+    const tx = await coreManager.listingAsset(args.listingParams, args.investmentData, args.documents);
+    const result = await tx.wait();
+    if (result.status === TxStatus.Success) {
+      console.log('Success')
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/models/newAssetModel.ts:146]: please remove logs or make more verbose - Success on what?)

File: api/src/features/buyingHistory/buyingHistory.service.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ export class BuyingHistoryService implements OnModuleInit {
       this.configService.get('PRC_URL'),
     );
     this.propertyFactoryContract = new ethers.Contract(
-      '0x6fCA3B71d7836cf9369f7a9D80d4F5BB5Ff0F2d1',
+      '0x8905a6cd5e5D653Dfc69C5DD5e16bC2bFA8CfEDF',
```
Comment (ruslanpalagin) [inline api/src/features/buyingHistory/buyingHistory.service.ts:27]: I made it configurable - will be in my next PR

File: contracts_arbitrum/hardhat.config.ts (hunk 1 above)
Comment (Space-zee) [reply contracts_arbitrum/hardhat.config.ts:26]: +

File: frontend/app/features/assets/models/newAssetModel.ts (hunk 3 above)
Comment (Space-zee) [reply frontend/app/features/assets/models/newAssetModel.ts:146]: +

**Review by ruslanpalagin — APPROVED** [review]

**Review by ruslanpalagin — APPROVED** [review]

**Pull request merged**